### ISSUE TITLE
Final Standings not shown after Finalize Tournament

### ISSUE THREAD

**Issue by kkoskelin** [coldlink/challongerv2#1]
Hey there - great app, thanks!
I've just installed v2.0.6 this evening (Samsung Galaxy S6) and reset an old "test" tournament which I can send you a link if you PM me.  It's a single-stage, double-elimination "Raingutter Regatta" tournament with only 5 participants (again, just for testing).
After resetting the tournament and adding in all the results, I finalized the tournament but the "Final Standings" screen didn't work.
- The standings button (bar graph) icon appeared
- The title "Final Standings" did not appear
- The content of the final standings did not appear
  This seemed to get fixed if I exited the tournament, refreshed and re-opened it and went to Final Standings.

Glad to answer any questions you have.

**Comment by coldlink** [comment]
Thanks for letting me know!

I'll see if I can replicate the issue myself using what you provided above. If not, I'll be in touch to see if its specific to your test tournament, or if it's something else.

Thanks

**Comment by coldlink** [comment]
I've reproduced the issue myself, will have a look at whats causing it. Thanks for finding it.

**Comment by kkoskelin** [comment]
👍  happy to re-test when addressed.

**Comment by coldlink** [comment]
I've attempted a fix which seems to be working at my end on my android devices. I've attached an apk with the fix. Will need to uninstall the old version and try out this apk.

This issue itself was to do with caching, when the tournament was completed it was for some reason returning the cached 'empty' version. By forcing a reload when finalising the tournament clears the cache, and shows the final standings tab correctly. There is probably a more elegant solution I'll look into before updating it on all platforms.

[bracketorv2-2.0.7-testing.apk.zip](https://github.com/coldlink/bracketorv2/files/305220/bracketorv2-2.0.7-testing.apk.zip)
